File: Modul 1/soal 8/PRAK108-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
#define PI 3.14

int main() {
    int putaran = 5;
    float total_jarak = 14; 
    float keliling, jari_jari;

    keliling = total_jarak / putaran;
    jari_jari = keliling / (2 * PI);

    printf("Diketahui : \n");
    printf("Pak Dengklek mengelilingi taman = %d putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n\n\n", total_jarak);
    printf("Jawaban : \n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jari_jari);
    
    return 0;
}

File: Modul 1/soal 8/PRAK108-2410817220018-ANGGRAENI DWI ZAHRA.py
import math

putaran = 5
jarak_tempuh = 14 

keliling_satu_putaran = jarak_tempuh / putaran
jari_jari = keliling_satu_putaran / (2 * math.pi)

print("Diketahui :")
print(f"Pak Dengklek mengelilingi taman = {putaran} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak_tempuh} Kilometer")
print()
print()
print("Jawaban :")
print(f"Jari-jari taman yang dikeliingi Pak Dengklek adalah {jari_jari:.2f} Kilometer")